### ISSUE TITLE
Downloading .genome files manually (when machine doesn't have mysql)

### ISSUE THREAD

**Issue by hdashnow** [MelbourneGenomics/cpipe#8]
The install.sh script checks for genome file /vlsci/VR0193/shared/cpipe/hg19/hg19.genome and tries to download it if it doesn't exist. This automatic download is only possible on machines with mysql (so failed on Merri, for example).

What is hg19.genome? And where do I download it manually?

I see from config.groovy that "Genome needed (currently) only if using expanded splice regions"

Yet the install script says:
WARNING: Cpipe will not operate correctly unless the /vlsci/VR0193/shared/cpipe/hg19/hg19.genome file is created

Might want to add some clarifying statements to the install script, and the documentation (it's not currently mentioned in the documentation PDF or the hg19 README)

**Comment by hdashnow** [comment]
So, appearently the .genome file is necessary. When I tried to run the pipeline without it, this command fails:

```
[hdashnow@merri analysis]$ python /vlsci/VR0193/shared/cpipe/pipeline/scripts/create_exon_bed.py   -c -s ../design/CARDIOM.bed /vlsci/VR0193/shared/cpipe/tools/annovar/humandb/hg19_refGene.txt ../design/CARDIOM.transcripts.txt - | /vlsci/VR0193/shared/cpipe/tools/bedtools/2.18.2/bin/bedtools slop -g /vlsci/VR0193/shared/cpipe/hg19/hg19.genome -b 2 -i - > ../design/CARDIOM.splice.bed;  python /vlsci/VR0193/shared/cpipe/pipeline/scripts/create_exon_bed.py   -c ../design/CARDIOM.bed /vlsci/VR0193/shared/cpipe/tools/annovar/humandb/hg19_refGene.txt ../design/CARDIOM.transcripts.txt ../design/CARDIOM.exons.bed
Error: The requested genome file (/vlsci/VR0193/shared/cpipe/hg19/hg19.genome) could not be opened. Exiting!

...

```

**Comment by ssadedin** [comment]
You're right - it used to be not required because we did not use a window
for splice variants by default. Now we do, so it's needed. I will update
the installer.

**Comment by hdashnow** [comment]
where can I download it?

**Comment by ssadedin** [comment]
Probably the best method is to download from UCSC using
the mysql method if possible though. Will look into whether we can do it
without mysql somehow. I will send you a file to use in the interim.
